Highlight/Highlighter.php

```diff
@@ -563,12 +563,49 @@ public function highlightAuto($code, $languageSubset = null)
      * setAutodetectLanguages will turn on autodetection for all supported
      * languages.
      *
+     * @param bool $include_aliase Specify whether language aliases
+     *      should be included as well.
+     *
      * @return array
      *      An array of language names (strings).
```

Comment:
While not related to your PR but since you're touching this section of the code, could you change `array` to `string[]` and get rid of `(strings)` from the sentence too, please?